packages/terra-folder-tree/src/subcomponents/FolderTreeItem.jsx

```diff
@@ -91,7 +103,10 @@ const FolderTreeItem = ({
       <li
         className={itemClassNames}
         role="treeitem"
+        aria-expanded={isFolder ? isExpanded : null}
         aria-selected={isSelected}
+        onClick={handleExpandCollapse}
+        onKeyDown={handleExpandCollapse}
```

Comment:
Aren't there specific keys that trigger this behavior?  The current logic appears to expand/collapse when any key is pressed.

Reply:
`onKeyDown` was added here to fix an eslint error, the actual keyboard logic will be added in UXPLATFORM-9776.

Reply:
The key down event would be part of the expand/collapse logic though.  You can handle the specific key for just this functionality without accounting for the keyboard navigation.